src/liballoc/alloc.rs

```diff
@@ -56,6 +56,21 @@ pub struct Global;
 /// # Safety
 ///
 /// See [`GlobalAlloc::alloc`].
+///
+/// # Examples
+///
+/// ```
+/// use std::alloc::{alloc, dealloc, Layout};
+///
+/// unsafe {
+///     let layout = Layout::from_size_align(2, 1).unwrap();
```

Comment:
`u16` generally has an alignment of 2 not 1. Using `Layout::new::<u16>()` would be better here though.